### PR TITLE
Help/documentation update

### PR DESCRIPTION
**Pull request by LilliJane**
This will remove a useless step, instead of asking the user to look by himself for the documentation, we provide it directly.

### CODE REVIEW

**Comment by LilliJane** [comment]
One minor change: I changed the `-v` to `-V` for the version check, because lots of people are used to have `-v` for `verbose`.

File: command/command.go [hunk 1]
```diff
@@ -119,7 +120,11 @@ func (c *command) Run(args []string) int {
 	}
 
 	if err := c.runFunc(args); err != nil {
-		if err == errShowHelp {
+		if strings.Contains(err.Error(), errShowHelp.Error()) {
```
Comment (advanderveer) [inline command/command.go]: The general approach here is really nice (using an error value to indicate we need to show help)!. I'm not a fan of the string comparison though, this is fragile. Take a look at the "cause" functionality of the errors pkg: https://github.com/pkg/errors#retrieving-the-cause-of-an-error.

**Comment by advanderveer** [comment]
I checked this out and really like the improvement, nice that it still says that it lacks arguments. About the version flag, I agree that i collides with the verbose flag. It is currently inconsistent thought as `nerd -v` still returns the version. If this is hard to fix I'm still up for keeping the change.

**Comment by LilliJane** [comment]
You're right about the `-v`, as it's a feature of the `mitchellh/cli`, I'll look if we can override it. But first I'll update the error handling

File: command/command.go (hunk 1 above)
Comment (LilliJane) [inline command/command.go]: Didn't know about this, it's updated now

**Comment by LilliJane** [comment]
last commit closes #145

**Comment by LilliJane** [comment]
And last commit closes #146

**Comment by LilliJane** [comment]
Close #139

**Pull request merged**